### DISCUSSION TITLE
The screen size changing issue in Windows 11

### DISCUSSION THREAD

**Discussion by furkanozturklab** [illixion/vscode-vibrancy-continued#144] [Q&A]
After installing the extension, the key for enlarging and shrinking VSCode doesn't work. Even after uninstalling the extension, the problem persists, requiring a complete reinstallation for it to work again. How can this issue be fixed?

**Comment by illixion** [comment]
Please check FAQ for uninstallation instructions. Minimize/maximize issue is tracked in https://github.com/illixion/vscode-vibrancy-continued/issues/140
